modules/apim_common/manifests/init.pp

```diff
@@ -45,9 +45,20 @@
   */
 
   # Copy JDK to Java distribution path
-  file { "jdk-distribution":
-    path   => "${java_home}.tar.gz",
-    source => "puppet:///modules/${module_name}/jdk/${jdk_name}.tar.gz",
+  if $pack_location == "local" {
+    file { "jdk-distribution":
+      path   => "${java_home}.tar.gz",
+      source => "puppet:///modules/${module_name}/jdk/${jdk_name}.tar.gz",
+      notify  => Exec["unpack-jdk"],
```

Comment:
Please format the code.

Reply:
Fixed https://github.com/wso2/puppet-apim/pull/128/commits/775a688e3eae96b3f6c013f85e1fed7bd19ae827